packages/vanilla/src/controls/RadioGroup.tsx

```diff
@@ -58,7 +58,7 @@ export const RadioGroup = ({
   const radioInput = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.input'), [contextStyles]);
   const radioLabel = useMemo(() => findStyleAsClassName(contextStyles)('control.radio.label'), [contextStyles]);
   const isValid = errors.length === 0;
-  const divClassNames = `validation  ${isValid ? classNames.description : 'validation_error' }`;
+  const divClassNames = `validation  ${isValid ? classNames.description : classNames.validationError }`;
```

Comment:
Do the same as in [the input control](https://github.com/eclipsesource/jsonforms/blob/master/packages/vanilla/src/controls/InputControl.tsx#L70)